pkg/mgrconfig/load.go

```diff
@@ -338,7 +339,9 @@ func ParseEnabledSyscalls(target *prog.Target, enabled, disabled []string) ([]in
 		}
 	}
 	for call := range syscalls {
-		if target.Syscalls[call].Attrs.Disabled {
+		if target.Syscalls[call].Attrs.Disabled ||
+			descriptionsMode == "manual" && target.Syscalls[call].Attrs.Auto ||
```

Comment:
If you use a string, we need to check that the value is actually one of the 3 allowed values. Otherwise it's too easy to make a typo.